packages/lesswrong/components/tagging/subforums/TagSubforumPage2.tsx

```diff
@@ -59,6 +59,7 @@ export const styles = (theme: ThemeType): JssStyles => ({
   },
   title: {
     ...theme.typography.headline,
+    fontFamily: theme.palette.fonts.sansSerifStack,
```

Comment:
Hmm I think I prefer the serif for this case, makes the page feel more official or something. But feel free to ignore me, this is Agnes's call :)